### DISCUSSION TITLE
using drought indices as stress gradients

### DISCUSSION THREAD

**Discussion by HaireLab** [HaireLab/community-stress-gradients#3] [General]
Thanks for the discussion today! Here are a few thoughts for your feedback.

Using an index like SPEI is not so much about using the actual values, but rather, identifying trends in the data. In that case, it doesn't seem important that the values are relativized but more about what trends are of interest and how to go about quantifying them.

Some examples would be: 
- shifts in mean and/or variance
- jumps in the data
- peaks and valleys
- outliers in a given period
- breaks, trend models

Then, the site-level stress metrics would be parameters derived from these quantities. I'll work on sharing some examples. In the meantime---what are your thoughts and reactions?

**Comment by megkrawchuk** [comment]
Seems like peaks and valley or outliers in a given period could be an interesting way to quantify potential critical "pinch points" of drought stress? At least as a hypothesis. I guess we could come up with a few different metrics as competing hypothesis for which integrative metrics best capture drought stress on community over time.

**Reply by HaireLab** [reply]
Almost ready with several options to consider--now switched from SPEI to Deficit, a growing season metric: Evaporative demand that is not met by available water and is thus an index of potential effects of drought stress on plants. This is also known as Climate Water Deficit, but I prefer the shortened name which can be abbreviated D!

**Reply by HaireLab** [reply]
Starting with summary statistics across years--the plots, below represent binned values for sample locations within our climate space. I chose the colors to bring out high, moderate and low ranges--moderate corresponding to locations where we think community interactions would be most important. Note: PC 1 = temperature, PC 2 = precipitation

**Reply by HaireLab** [reply]
Another note: these data are 4 km values for Def from TerraClimate

**Comment by HaireLab** [comment]
![def max](https://user-images.githubusercontent.com/68163030/119701183-11b25f00-be22-11eb-83a1-f681bcd3cb48.png)
![def mean](https://user-images.githubusercontent.com/68163030/119701253-255dc580-be22-11eb-8fea-5af0cc23138b.png)
![def q75](https://user-images.githubusercontent.com/68163030/119701270-2989e300-be22-11eb-857d-c1aa92ea2ae7.png)
![def q90](https://user-images.githubusercontent.com/68163030/119701291-2e4e9700-be22-11eb-99c6-a6fe7bc3fb14.png)
![def var](https://user-images.githubusercontent.com/68163030/119701307-31498780-be22-11eb-8507-f1c63c88f2c3.png)

**Reply by megkrawchuk** [reply]
Thanks Sandra! Interesting that the major patterns seem to be consistent among all versions of the metric.

**Comment by HaireLab** [comment]
**More plots, summarizing Def values in post-fire years for each study site**

![def maxbox](https://user-images.githubusercontent.com/68163030/119719154-71b30080-be36-11eb-96a7-afc9d53146c8.png)
![def meanbox](https://user-images.githubusercontent.com/68163030/119719169-74adf100-be36-11eb-816f-1dddb3527e58.png)
![def varbox](https://user-images.githubusercontent.com/68163030/119719177-77a8e180-be36-11eb-943c-a0283ba2a113.png)
![def q75box](https://user-images.githubusercontent.com/68163030/119719191-7b3c6880-be36-11eb-9bfa-f30d01ba5838.png)
![def q90box](https://user-images.githubusercontent.com/68163030/119719199-7e375900-be36-11eb-8a1c-a7a7be3a5e56.png)

**Reply by megkrawchuk** [reply]
These are helpful for illustrating the range of variability within fires and also how they all line up against one another. Would it be a simple talks to order them so that the x-axis has the fires arranged in order of increasing median? I think that'd be helpful for me to think about this more clearly, to put them on that initial "stress" gradient (at the fire level) and also see how much overlap there is among sample points from different fires.

Are you starting to think about one or two of these D metrics as candidates for an initial model run?

**Reply by HaireLab** [reply]
Good idea. I can easily rearrange. I also noticed I need to figure out what's going on with Cerro Grande--it's the one site with very narrow range.
We can consider these metrics for modeling, but I also want to explore some of the time series parameters e.g., peaks, jumps, shifts, outliers--there are options for using kernels across the time series that would be fun to try!

**Reply by HaireLab** [reply]
On second thought--not really helpful to rearrange--couldn't get any clearer view in my tries at this. So I went with plotting the values for each site/burn separately. What do you think of this perspective? (also in ppt on google drive)
Think I like the variance...
![Dvar_facets](https://user-images.githubusercontent.com/68163030/122399833-24afdf00-cf49-11eb-9811-95d050d72532.png)

**Comment by megkrawchuk** [comment]
Hi Sandra! This is helpful to see the individual plots/points. Were you also thinking of continuing your idea of peaks, jumps etc. along the lines of critical filters or pinch points? The variability seems like a nice start, and I think we also need a metric to represent the gradient itself, in addition to how much variability they experience?
Curious to hear what you think!

**Reply by HaireLab** [reply]
Yes, still working on the peaks and so on. But the variance of the values across time does provide an overall metric for the gradient that tells the degree of fluctuation that each location was experiencing. I also like the quantile values (75 and 90 are in the data) because these give more information on the distribution of values through time for each field plot location.  The peaks and valley metrics (coming soon) might be more difficult to interpret. But we shall see....
